examples/plot_pp_study.py

```diff
@@ -0,0 +1,362 @@
+
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
+# Licensed under the MIT License.
+
+"""
+================================
+Predictive Policing Case Study
+================================
+"""
+# %%
+#
+#
+# Contents
+# ========
+#
+# 1. Overview
+# 2. Background
+# 3. Fairlearn Results
+# 4. Problem area & Sociotechnical context
+# 5. Conclusion
+#
+# Overview
+# --------
+#
+# In this document we outline a sample sociotechnical case study - using a
```

Comment:
I would feel more comfortable if we could mention the purpose of this study from the start. I think we should express that we do not endorse predictive policing and point to the extensive research on the topic that outlines related issues.